### ISSUE TITLE
Navigate into multi gallery

### ISSUE THREAD

**Issue by ben4523** [sachinchoolur/lightGallery#565]
Hy,
I would like to know if a have some solution to navigate with the button back and next into multi gallery in the same page.

if i have 3 gallery in the page i would like to navigate with button next in the 3 gallery.
Thanks

**Comment by sachinchoolur** [comment]
Hi @ben4523 ,
I'm sorry, but I did not understand your question properly. Could you rephrase it?

**Comment by ben4523** [comment]
Yes sure, 
Immagine we have 3 gallery in one page, i would like to navigate with the previous and next button into all gallery. if we are at the last picture of the first gallery when i click to the next button i go to the first picture of the second gallery. As well as continuing.
Thanks @sachinchoolur

**Comment by sachinchoolur** [comment]
Yes, It is possible. Here is the demo - http://codepen.io/anon/pen/zZpJwx

Let me know if you have any queries

**Comment by ben4523** [comment]
You are the boss, it that. 
another question it's possible to create the pagination system for the thumbnail ? you see in my picture.
Thanks you very much @sachinchoolur

**Comment by sachinchoolur** [comment]
You can use any of the content sliders available on the web. Here is the demo - http://sachinchoolur.github.io/lightslider/examples.html
